PullRequestCreated/ReadMe.md

```diff
@@ -7,6 +7,10 @@ This action will take the first non-`Sub-task` ticket and use it as a parent for
 - if the parent is not an `Epic`, the created ticket will be a `Sub-task`
 - if there is no parent, the created ticket will be a `Task`
 
+This action will also update the PR title and description to mention the right Jira tickets.
+The action will link all mentioned tickets in the title into the description.
+When creating a ticket, the action will add it to the PR title.
```

Comment:
This should be before the point about updating the description, to keep them in (chrono)logical order
```suggestion
When creating a new Jira issue, the action will add the new issue ID to the PR title.
```